<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSvgSelector</class>
 <widget class="QWidget" name="WidgetSvgSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <widget class="QLabel" name="mImagesLabel">
     <property name="text">
      <string>SVG Images</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mGroupsLabel">
     <property name="text">
      <string>SVG Groups</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeView" name="mGroupsTreeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>2</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QListView" name="mImagesListView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="layoutMode">
      <enum>QListView::Batched</enum>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="gridSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="mFileLayout">
     <item>
      <widget class="QLineEdit" name="mFileLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="mFilePushButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mRelativePathChkBx">
       <property name="toolTip">
        <string>Generated path will be relative to current SVG search directories or to Project file</string>
       </property>
       <property name="text">
        <string>Relative path</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mImagesListView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
